com.microsoft.mrtk.input/Interactors/InteractorVisuals/BaseReticleVisual.cs

```diff
@@ -25,7 +25,25 @@ public class BaseReticleVisual : MonoBehaviour, IXRCustomReticleProvider
 
         public GameObject Reticle => customReticleAttached ? customReticle : baseReticle;
 
-        #region IXRCustomReticleProvider Implementation
+        private IVariableReticle variableReticle;
+
+        public IVariableReticle VariableReticle
+        {
+            get
+            {
+                if (variableReticle == null)
+                {
+                    variableReticle = Reticle.GetComponent<IVariableReticle>();
+                }
+                else
+                {
+                    Debug.LogWarning("No IVariableReticle found for the Reticle Visual.", this);
+                }
```

Comment:
I think this `else` case is in the wrong spot. Right now, it'll hit if `VariableReticle` is called and `variableReticle` is non-null

Reply:
Woops! removed.